README.md

```diff
@@ -1,9 +1,34 @@
 
-# CFPB Front-End Team
+# CFPB front-end team
 
 > A repository for the discussion and publication of the CFPB front-end team standards.
 
 ----
+## Standards and styles
+
+Over the coming months we'll be establishing standards and style guides for working with the front-end of CFPB sites and applications. As we do so, we'll be discussing these standards in the [issues](https://github.com/cfpb/front-end/issues) of this repository and posting final documents here, with the final goal of building something similar to the [CFPB Design Manual](http://cfpb.github.io/design-manual/). As a team, we'll be sharing responsibility for this process and will initially focus on our core competencies:
```

Comment:
Please employ "semantic line breaks" here. I ask this because horizontal scrolling is the devil. (At least GitHub diffs don't do that anymore, but it can still happen in your text editor, depending on one's configuration.)

But seriously, the real reason is this: having multiple lines makes for easier diffing if you change a tiny part of a whole paragraph. [See here for more.](https://scott.mn/2014/02/21/semantic_linewrapping/)